compiler/env/FrontEnd.hpp

```diff
@@ -181,6 +182,9 @@ class TR_FrontEnd : public TR_Uncopyable
    virtual TR_YesNoMaybe isInstanceOf(TR_OpaqueClassBlock *instanceClass, TR_OpaqueClassBlock * castClass, bool instanceIsFixed, bool castIsFixed = true, bool callSiteVettedForAOT=false);
    virtual bool isUnloadAssumptionRequired(TR_OpaqueClassBlock *, TR_ResolvedMethod *);
 
+   // returns true if list of classes contains less then two (zero or one) concrete classes
+   virtual bool noMultipleConcreteClasses(List<TR_PersistentClassInfo>* subClasses);
```

Comment:
A few comments:
1. Can the comment be written in Doxygen format, please? Also, the comment should clearly explain what is meant by "concrete class." Specifically, I was confused about why the function couldn't just be `subClasses->getSize() < 2` until I looked at the code.
2. The name `noMultipleConcreteClasses` seems very unclear to me. Have you considered a name like `containesZeroOrOneConcreteClass`?
3. Also, as explained in the comment at the top of this file, the FrontEnd interface is deprecated. Is there a reason this query can't be in `ClassEnv` or some other location?

Reply:
1. I have added Oxygen format comment for the function prototype. First commit in OMR, so please let me know if that is the correct format. 
2. Could I stay with this name `noMultipleConcreteClasses` ? There is a [PR](https://github.com/eclipse/openj9/pull/8933) in OpenJ9 that has been merged that uses this name, so just to be consistent. The comment also addressed the purpose of this function, which should be clear?
3. Looking into this item right now, might need your suggestion to port this function to other location.

Reply:
I have no preference for the name of the method, so a change to `containesZeroOrOneConcreteClass` can be considered. Obviously openj9 has to change as well, but that has to happen anyway since the new function has moved from frontend to ClassEnv